internal/cephfs/fsjournal.go

```diff
@@ -44,15 +56,15 @@ because, the order of omap creation and deletion are inverse of each other, and
 request name lock, and hence any stale omaps are leftovers from incomplete transactions and are
 hence safe to garbage collect.
 */
-func checkVolExists(ctx context.Context, volOptions *volumeOptions, secret map[string]string) (*volumeIdentifier, error) {
+// nolint:gocognit:gocyclo // TODO: reduce complexity
+func checkVolExists(ctx context.Context,
+	volOptions,
+	parentVolOpt *volumeOptions,
+
```

Comment:
why this empty line?

Reply:
to distuinguish the param types in the arg list.